package.json

```diff
@@ -21,6 +21,7 @@
     "package": "./package.sh",
     "security-scan": "git secrets --scan && git log -p | scanrepo && snyk test",
     "test": "npm run compile && jasmine",
+    "integration": "tsc integration/templates/generateTemplates.ts && node integration/templates/generateTemplates",
```

Comment:
Are we just trying to execute a TS file? If so we can use `ts-node integration/templates/generateTemplates.ts` to achieve this directly.

Reply:
yea, that works